### ISSUE TITLE
Radio buttons no longer have visible focus

### ISSUE THREAD

**Issue by MRayermannMSFT** [Microsoft/AzureStorageExplorer#237]
**Storage Explorer Version**: 1.2.0
**Platform**: Windows
**Architecture**: win32
**Regression From**: Unknown

#### Steps to Reproduce: ####
1. Go to StorageExplorer/coolstorageencrypted/encrypted
2. Navigate into .vscode
3. Double click on tasks.json
4. Observe the unsupported filename dialog appearing
![image](https://user-images.githubusercontent.com/21112697/40808590-85186bda-64dc-11e8-944a-06302fda8a8d.png)
5. Tab to the radio buttons

#### Expected Experience: ####
I can see that the radio buttons have focus

#### Actual Experience: ####
I cannot see that the radio buttons have focus. I also observe this in the Connect and Generate SAS dialog
